_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+it evaluated was to reduce the subsidy by 20% every time the network's
+proof of work security doubles (measured by block-creation difficulty).
+
+*[video][vid subsidy], [transcript][tx subsidy], soft fork*
+
+### Forward blocks: on-chain capacity increases without a hard fork
+
+*Research by Mark Friedenbach*
+
+One well-known method for soft forking an increase in the Bitcoin block
+size is extension blocks---a data structure that's invisible to nodes
+that haven't upgraded to the soft fork and so is not subject to their
+historic limits on block size.  By itself, this is an undesirable method
+for increasing block size because preventing old nodes from seeing the
+transactions in the extension block also prevents them from being able
+to enforce any other consensus rules on those transactions---such as
+rules that prevent a malicious user from spending other users' bitcoins
+or from creating more bitcoins than allowed by the 21 million bitcoin
+subsidy schedule.
+
+However, one doesn't need to increase block size to increase the amount
+of data that can be added to the block chain per minute---it's also
+possible to increase capacity by increasing the frequency of blocks
+(reducing the average time between blocks).  A method for gaming
+Bitcoin's difficulty adjustment algorithm---called a time-warp
+attack---is well-known among experts and has been used successfully in
+demonstration attacks against Bitcoin's testnet and real attacks against
+altcoins.  (Note: although Bitcoin is technically vulnerable to this
+attack, it'd be a slow attack that would give the userbase a significant
+amount of time to respond.)  By itself, increasing block frequency is
+also an undesirable method for increasing capacity because shorter block
+intervals increase the effectiveness of miners with large amounts of
+hashrate and so is likely to increase mining centralization.[^freq-pow-waste]
+
+Perhaps disproving the saying that "two wrongs don't make a right," this
+talk describes a novel way of combining extension blocks and the
+time-warp attack to allow both upgraded nodes and old nodes to gain the
+same capacity increase and see all the same transactions for validation
+while simultaneously slightly reducing mining centralization risk.
+Upgraded nodes would validate one or more extension blocks providing
+additional block space with a 15 minute average interval, but they'd
+also restrict the time stamps in legacy blocks to ensure a permanent
+(but limited) time warp attack increased the frequency of legacy blocks
+enough to allow them to include the same transactions that previously
+appeared in the extension blocks.
+
+*[video][vid forward blocks], [transcript][tx forward blocks],
+[paper][paper forward blocks], soft fork*
+
+### Compact multi-signatures for smaller blockchains
+
+*Research by Dan Boneh, Manu Drijvers and Gregory Neven*
+
+This talk describes an alternative to the Bellare-Neven-based Schnorr
```

Comment:
Perhaps rephrase this a bit. At the moment it sounds like Mihir Bellare and Gregory Neven collaborated with a cryptographer named 'based' to create a signature scheme.